recipe/build.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+# Get an updated config.sub and config.guess
+cp $BUILD_PREFIX/share/gnuconfig/config.* ./config
```

Comment:
@183amir: the problem is here.  The 1.0.2 checkout of blitz++ has a `config/` directory which points to **broken** symlinks.  The `cp` command fails, which breaks all builds.  All links in that directory are wrong.  I think the best is to remove this directory, and then re-create an empty version of it.  The copy command will then work.  I don't have the rights to push to the affected repository, unfortunately.  Here is the patch:

```diff
diff --git a/recipe/build.sh b/recipe/build.sh
index d70b464..e19e09e 100644
--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -1,5 +1,7 @@
 #!/bin/bash
 # Get an updated config.sub and config.guess
+rm -rf ./config
+mkdir ./config
 cp $BUILD_PREFIX/share/gnuconfig/config.* ./config
 set -ex
```